### DISCUSSION TITLE
Unauthrorized error when using Catalog API

### DISCUSSION THREAD

**Discussion by jonathan-villanueva** [saleweaver/python-amazon-sp-api#95] [General]
Hello,
I get the error: sp_api.base.exceptions.SellingApiForbiddenException: [{'message': 'Access to requested resource is denied.', 'code': 'Unauthorized', 'details': ''}]

when I use the Catalog endpoint:
resp = Catalog(credentials=credentials).list_items(MarketplaceId=Marketplaces.US.marketplace_id, UPC='885909918164')

However I am using the same credentials and I do not have any problems using the Fees module.

**Comment by yberezkin** [comment]
Can you please, check in Seller Central what access has been granted to your credentials?

**Reply by jonathan-villanueva** [reply]
Actually, I solved the problem by granting access to additional features on Seller Central. I guess it just took some time to take effect. Thank you!

**Reply by yberezkin** [reply]
Sure, no problem 😃
